### PR TITLE
VIMC-4619: Add orderly_info to return info about report run

### PR DESCRIPTION
**Pull request by r-ash**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# orderly 1.2.39
+
+* New function `orderly_info` which returns details from failed and succeeded report runs (VIMC-4619)
```
Comment (richfitz) [inline NEWS.md:3]:
```suggestion
* New function `orderly_info` which returns details from report runs, both successful and failed (VIMC-4619)
```

File: R/info.R [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    info <- readRDS(success_rds)
+    success <- TRUE
+  } else {
+    fail_rds <- path_orderly_fail_rds(report)
+    if (file.exists(fail_rds)) {
+      info <- readRDS(fail_rds)
+      success <- FALSE
+    } else {
+      stop(sprintf(
+        "Failed to retrieve info for report %s:%s, rds does not exists",
```
Comment (richfitz) [inline R/info.R:32]:
```suggestion
        "Failed to retrieve info for report %s:%s, rds does not exist",
```

File: R/paths.R [hunk 3]
```diff
@@ -52,6 +52,10 @@ path_orderly_workflow_dir <- function(path, workflow_name) {
   file.path(path, "workflows", paste0(workflow_name, ".yml"))
 }
 
+path_orderly_log <- function(path) {
```
Comment (richfitz) [inline R/paths.R:55]: amazed we did not have this!

File: R/info.R [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+##'
+##' This will return info from either successful or failed reports. It will
+##' look for the report with `id` in archive first and then look in drafts
+##' if it can't be found from archive.
+##'
+##' @param id The report ID
+##' @param name The name of the report
+##' @inheritParams orderly_list
+##'
+##' @return Info from report run
```
Comment (richfitz) [inline R/info.R:11]: Do we want to advertise the exact format is subject to change? If not do you want to document it?

**Review by richfitz — CHANGES_REQUESTED** [review]
This is great

File: R/info.R (hunk 4 above)
Comment (r-ash) [reply R/info.R]: Thanks, added a note saying it is subject to change. And added some details too about what it returns at the moment.

**Review by richfitz — APPROVED** [review]

**Pull request merged**